### DISCUSSION TITLE
Temporarily Shutdown the Cluster

### DISCUSSION THREAD

**Discussion by Gerthum** [strimzi/strimzi-kafka-operator#10849] [Q&A] [answered]
Hi Team

We have a 3x broker cluster and would like to Snapshot the disks in case of a failure. What would be the best way to shut it down temporarily in order to take a Snapshot of the PVs. Once complete it can start-up again. This is only a passive cluster so it doesn't have any consumers ect.  connected to it. 

Thanks in advance.

**Comment by scholzj** [comment, marked as answer]
You can do the following:
1. Stop the Strimzi reconciliation using the pause-reconciliation annotation (check the docs for the exact format) on the Kafka CR
2. Delete the StrimziPodSet(s) matching the nodes that you want to stop
3. Snapshot your disks
4. Unpause the reconciliation by removing the annotation

**Reply by Gerthum** [reply]
Thank you will test this approach and report back if I have any issues.
